QuickBytes/src/main/java/com/quickbytes/controller/ComboController.java

```diff
@@ -1,13 +1,87 @@
-package com.quickbytes.controller;
+	package com.springboot.backend.controller;
```

Comment:
we should not rename packages, this will mess things up.

Reply:
This error needs to be fixed in all files